packages/renderer/src/lib/feedback/feedbackForms/DevelopersFeedback.spec.ts

```diff
@@ -0,0 +1,174 @@
+/**********************************************************************
+ * Copyright (C) 2023-2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import '@testing-library/jest-dom/vitest';
+
+import { fireEvent, render, screen } from '@testing-library/svelte';
+import userEvent from '@testing-library/user-event';
+import { beforeAll, expect, test, vi } from 'vitest';
+
+import DevelopersFeedback from './DevelopersFeedback.svelte';
+
+beforeAll(() => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (window as any).openExternal = vi.fn();
+  (window as any).telemetryTrack = vi.fn();
+  (window as any).sendFeedback = vi.fn();
```

Comment:
nitpick: can use `Object.defineProperty(global, 'window',`  pattern so no need to use explicit-any

Reply:
for some reason, using the `Object.defineProperty(global, 'window'` pattern caused the tests to fail (rendered components from previous tests "leaked" into the tests after them), so I used `Object.defineProperty(window, '<method name>'` for each one